{"name": "test_correct_auth_after_bad", "status": "passed", "steps": [{"name": "Некорректная авторизация", "status": "passed", "start": 1685358200668, "stop": 1685358202154}, {"name": "Авторизация", "status": "passed", "start": 1685358202154, "stop": 1685358204194}, {"name": "Выход из учетной записи", "status": "passed", "start": 1685358204195, "stop": 1685358204558}], "attachments": [{"name": "stdout", "source": "c114b1e2-c463-4d18-b14e-38d9666a1e7c-attachment.txt", "type": "text/plain"}], "start": 1685358199298, "stop": 1685358204559, "uuid": "7c94baeb-7941-4221-823f-87ef3744cc5f", "historyId": "0ee9b0362e4f021167d51cd46a2d43a1", "testCaseId": "0ee9b0362e4f021167d51cd46a2d43a1", "fullName": "test_test#test_correct_auth_after_bad", "labels": [{"name": "suite", "value": "test_test"}, {"name": "host", "value": "AfanasievA"}, {"name": "thread", "value": "11400-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "test_test"}]}